File: blog_vue/src/model/article/Article.ts
export default class Article {
  /**主键 */
  id: number;
  /**标题 */
  title: string;
  /**头像 */
  icon: string;
  /**内容 */
  body: string;
  /**简介 */
  synopsis: string;
  /**阅读量 */
  views: number;
  /**点赞量 */
  likeCount: number;
  /**是否私有 */
  ifPrivate: number;
  /**创建时间 */
  creationTime: string;
  /**修改时间 */
  modifyTime: string;

  constructor() {
    this.id = -1;
    this.title = "";
    this.icon = "";
    this.body = "";
    this.synopsis = "";
    this.views = 0;
    this.likeCount = 0;
    this.ifPrivate = -1;
    this.creationTime = "";
    this.modifyTime = "";
  }
}

File: blog_springBoot/src/main/java/com/synergism/blog/core/article/entity/Article.java
package com.synergism.blog.core.article.entity;

import com.baomidou.mybatisplus.annotation.FieldFill;
import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.util.Date;

import com.fasterxml.jackson.annotation.JsonFormat;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

/**
 * <p>
 * 文章表
 * </p>
 *
 * @author Synergism
 * @since 2022-10-07 10:07:37
 */
@Getter
@Setter
@TableName("article")
@ApiModel(value = "Article对象", description = "文章表")
public class Article {

    @ApiModelProperty("主键")
    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    @ApiModelProperty("头像")
    @TableField("icon")
    private String icon;

    @ApiModelProperty("标题")
    @TableField("title")
    private String title;

    @ApiModelProperty("正文")
    @TableField("body")
    private String body;

    @ApiModelProperty("摘要")
    @TableField("synopsis")
    private String synopsis;

    @ApiModelProperty("浏览量")
    @TableField("views")
    private Long views;

    @ApiModelProperty("点赞数")
    @TableField("like_count")
    private Long likeCount;

    @ApiModelProperty("是否私有")
    @TableField("if_private")
    private int ifPrivate;

    @JsonFormat(timezone = "GMT+8", pattern = "yyyy-MM-dd HH:mm:ss")
    @ApiModelProperty("创建时间")
    @TableField("creation_time")
    private Date creationTime;

    @JsonFormat(timezone = "GMT+8", pattern = "yyyy-MM-dd HH:mm:ss")
    @ApiModelProperty("修改时间")
    @TableField(value = "modify_time", fill = FieldFill.INSERT_UPDATE)
    private Date modifyTime;


}
